src/stylesheet/global.css

```diff
@@ -78,6 +82,18 @@ html, body, #__next {
     color: var(--black);
 }
 
+/* Arrow link style classes */
+
+.blue-arrow {
+    color: var(--blue);
+}
+
+.aqua-arrow {
+    color: var(--transform-aqua);
+}
+
+/* Arrow link style classes */
+
```

Comment:
There is no need to leave such comments. You can replace them by more meaningful class names. This is a modification to `.arrow`, so following BEM we would write something like `.arrow_brue-arrow` and `.arrow_aqua-arrow`